src/js/common/schemaform/actions.js

```diff
@@ -81,15 +82,22 @@ export function submitForm(formConfig, form) {
         return res.json();
       }
 
-      return Promise.reject(new Error(res.statusText));
+      return Promise.reject(new Error(`vets_server_error: ${res.statusText}`));
```

Comment:
Correct. You have get an http response code to get in here. Otherwise, fetch will throw an error. I've been testing this by shutting down my vets-api instance (which causes a connection refused error).